libs/accounts/src/lib/accounts-table.tsx

```diff
@@ -186,115 +204,113 @@ export const AccountTable = forwardRef<AgGridReact, AccountTableProps>(
             field="available"
             type="rightAligned"
             headerTooltip={t(
-              'This is the amount of collateral available in your general account.'
+              'Deposited on the network, but not allocated to a market. Free for use in new markets or orders.'
             )}
-            valueGetter={({
-              data,
-            }: VegaValueGetterParams<AccountFields, 'available'>) => {
-              return !data?.available
-                ? undefined
-                : toBigNum(data.available, data.asset.decimals).toNumber();
-            }}
-            valueFormatter={({
-              data,
-            }: VegaValueFormatterParams<AccountFields, 'available'>) =>
-              data &&
-              data.asset &&
-              isNumeric(data.available) &&
-              addDecimalsFormatNumber(data.available, data.asset.decimals)
-            }
-            maxWidth={300}
             cellRenderer={({
+              value,
               data,
-              node,
             }: VegaICellRendererParams<AccountFields, 'available'>) => {
-              const valueFormatted =
-                data &&
-                data.asset &&
-                isNumeric(data.available) &&
-                addDecimalsFormatNumber(data.available, data.asset.decimals);
-              return node.rowPinned ? (
-                <CenteredGridCellWrapper className="h-[30px] justify-end">
-                  {valueFormatted}
-                </CenteredGridCellWrapper>
-              ) : (
-                valueFormatted
+              const percentageUsed = percentageValue(data?.used, data?.total);
+
+              return (
+                <span className={colorClass(percentageUsed, true)}>
+                  {formatWithAssetDecimals(data, value)}
+                </span>
               );
             }}
           />
           <AgGridColumn
-            colId="breakdown"
-            headerName=""
-            sortable={false}
-            minWidth={200}
+            headerName={t('Total')}
             type="rightAligned"
-            cellRenderer={({
+            field="total"
+            headerTooltip={t(
+              'The full amount associated with this key. Total of used and available collateral.'
```

Comment:
Associated is a specific status for VEGA tokens in order to stake - is it the same meaning here?